<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DreamTeamTrad - FautesCheck</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/ressource/DreamteamLogo.ico</normaloff>:/ressource/DreamteamLogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_1" stretch="0,1">
      <property name="spacing">
       <number>17</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_1">
        <property name="title">
         <string>Choix du jeu et du google sheet</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_1_1" stretch="1,2">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_1_1_1">
             <property name="spacing">
              <number>16</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBox_game">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_1_1_1_1">
               <item>
                <widget class="QPushButton" name="pushButton_gameDictionary">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Dictionnaire termes du jeu</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_1_1_1_1">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_1_1_2">
             <property name="spacing">
              <number>16</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEdit_urlSheet">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>41</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>41</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="placeholderText">
                <string>mettre ici URL google sheet à traiter </string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_1_1_2_1">
               <property name="spacing">
                <number>13</number>
               </property>
               <property name="leftMargin">
                <number>18</number>
               </property>
               <item>
                <widget class="QLabel" name="label_frenchColumn">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Colonne du français :</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_frenchColumn">
                 <property name="minimumSize">
                  <size>
                   <width>41</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>41</width>
                   <height>41</height>
                  </size>
                 </property>
                 <property name="maxLength">
                  <number>2</number>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_1_1_2_1">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_sheetOpened">
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>name_sheet_opened</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_1_1_2_1_bis">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_1_1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_1_2" stretch="0,5">
        <property name="spacing">
         <number>24</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_1_2_1" stretch="0,0,0,2">
          <item>
           <widget class="QGroupBox" name="groupBox_1_2_1">
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>420</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>320</width>
              <height>420</height>
             </size>
            </property>
            <property name="title">
             <string>Mise à jour des fautes trouvées</string>
            </property>
            <widget class="QPushButton" name="pushButton_method_1">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>40</y>
               <width>141</width>
               <height>41</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Teste les caractères autorisés dans le jeu et si les mots existent. Rapide. Utilisez ça d'abord pour virer les termes spéciaux du jeu.</string>
             </property>
             <property name="text">
              <string>Mise à jour &quot;dictionnaire&quot;</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_method_2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>90</y>
               <width>141</width>
               <height>41</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Vérificateur de OpenOffice. Peut analyser les phrases. Un peu lent.</string>
             </property>
             <property name="text">
              <string>Mise à jour &quot;LanguageTool&quot;</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_lastUdate_2">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>90</y>
               <width>121</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>last_update</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_lastUdate_1">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>40</y>
               <width>121</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>last_update</string>
             </property>
            </widget>
            <widget class="QGroupBox" name="groupBox_1_2_1_1">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>140</y>
               <width>301</width>
               <height>271</height>
              </rect>
             </property>
             <property name="title">
              <string>Recherche</string>
             </property>
             <widget class="QPushButton" name="pushButton_method_search">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>30</y>
                <width>81</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>Rechercher</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_search">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>191</width>
                <height>41</height>
               </rect>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_search_result">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>120</y>
                <width>251</width>
                <height>41</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_method_replace">
              <property name="geometry">
               <rect>
                <x>110</x>
                <y>220</y>
                <width>81</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string>Remplacer</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_replace_1">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>75</y>
                <width>281</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Dans les résultats de la recherche, remplacer </string>
              </property>
             </widget>
             <widget class="QLineEdit" name="lineEdit_replace">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>170</y>
                <width>251</width>
                <height>41</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_replace_2">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>170</y>
                <width>41</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>par</string>
              </property>
             </widget>
            </widget>
            <widget class="QLabel" name="label_updateFolder">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>10</y>
               <width>121</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>last_update</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_1_2_1_1">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_uploadSpecificWords">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Pour limiter les appels à l'API de Google Sheet, uploader plusieurs termes d'un coup.</string>
              </property>
              <property name="text">
               <string>Pas de termes à uploader</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_1_2_1_1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_1_2_1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_result">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_1">
           <attribute name="title">
            <string>Dictionnaire</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_1">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>15</number>
              </attribute>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>fichier</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ligne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>faute</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>LanguageTool</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_2">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>15</number>
              </attribute>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>fichier</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ligne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>faute</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Recherche</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget_3">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>15</number>
              </attribute>
              <row>
               <property name="text">
                <string>1</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>fichier</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ligne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>faute</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_gameDictionary</tabstop>
  <tabstop>lineEdit_frenchColumn</tabstop>
  <tabstop>pushButton_method_1</tabstop>
  <tabstop>pushButton_method_2</tabstop>
  <tabstop>tabWidget_result</tabstop>
  <tabstop>tableWidget_1</tabstop>
  <tabstop>tableWidget_2</tabstop>
 </tabstops>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
